image_writer.py

```diff
@@ -0,0 +1,80 @@
+"""Takes in image text (top text and bottom text) and an image, and returns an image with text written on it in meme
+style.
+"""
+
+from PIL import Image, ImageDraw, ImageFont
+from random import randint
+
+
+def write_text_to_image(text_top, text_bottom, image_name):
+    """Main function. Writes inputted text to image in 'meme' style"""
+    image = _import_image(image_name)
+    # processed_image = _process_image
+    font, text_color, spacing = _determine_text_attributes()
+    draw = ImageDraw.Draw(image)
+    text_top_x, text_top_y, text_bottom_x, text_bottom_y = _determine_text_placement(draw, image, font, text_top,
+                                                                                     text_bottom)
+    _draw_image(draw, text_top, font, text_color, text_top_x, text_top_y, spacing)
+    _draw_image(draw, text_bottom, font, text_color, text_bottom_x, text_bottom_y, spacing)
+    _save_image(image, image_name)
+
+
+def _import_image(image_name):
+    """Imports selected image."""
+    return Image.open(image_name)
+
+
+# def _process_image(image):
+#     """Eventually will hold randomized image processing.
+#     """
+#     return image
+
+
+def _determine_text_attributes():
+    """Sets attributes for proper 'meme' style text"""
+    font = ImageFont.truetype('impact.ttf', 60)
+    text_color = (255, 255, 255)
+    spacing = 10
+    return font, text_color, spacing
```

Comment:
If this function is always returning constants, have them be CONSTS at the top of this module.